service/resource/deployment/master_deployment.go

```diff
@@ -5,14 +5,38 @@ import (
 	"path/filepath"
 
 	"github.com/giantswarm/kvmtpr"
-	"github.com/giantswarm/microerror"
 	apismetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	apiv1 "k8s.io/client-go/pkg/api/v1"
 	extensionsv1 "k8s.io/client-go/pkg/apis/extensions/v1beta1"
 
 	"github.com/giantswarm/kvm-operator/service/key"
 )
 
+func newEtcdDataVolume(customObject kvmtpr.CustomObject) apiv1.VolumeSource {
+	storageType := key.StorageType(customObject)
+
+	etcdVolume := apiv1.Volume{
+		Name: "etcd-data",
+		VolumeSource: apiv1.VolumeSource{
+			HostPath: &apiv1.HostPathVolumeSource{
+				Path: key.MasterHostPathVolumeDir(key.ClusterID(customObject), key.VMNumber(i)),
+			},
+		},
+	}
```

Comment:
@kopiczko had something against such defaulting. I am kind of with him. There should be a proper migration for the TPOs. I understand that you want to fix an incident though. So lets go with this and sort it properly afterwards. We only have to find a driver. And no, I will not drive this. <3